scenarios/zoo_intersection/agent_prefabs.py

```diff
@@ -17,9 +18,9 @@ def demo_agent_callable(target_prefix=None, interface=None):
 
 register(
     locator="zoo-agent1-v0",
-    entry_point="smarts.core.agent:AgentSpec",
+    entry_point="smarts.zoo.agent_spec:AgentSpec",
     # Also works:
-    # entry_point=smarts.core.agent.AgentSpec
+    # entry_point=smarts.zoo.agent_spec.AgentSpec
```

Comment:
While these work, I am surprised this is how it is set up. I believe the entrypoint should be `demo_agent_callable`.